packages/app/src/specs/InlineSpecList.cy.tsx

```diff
@@ -207,18 +190,43 @@ describe('InlineSpecList', () => {
   })
 
   describe('Run all Specs', () => {
+    const hoverRunAllSpecs = (directory?: string, specNumber?: number) => {
```

Comment:
This is only called in two places and both `directory` and `specNumber` are always defined. We can remove the `?`. It also means line 199 is never executed.